src/components/Button/Button.js

```diff
@@ -1,27 +1,43 @@
 // packages
 import React from 'react';
 import PropTypes from 'prop-types';
+import styled from 'styled-components';
 
-const Button = ({clickData, color, onClick, text, type, width}) => {
-  const styles = {
-    width,
-    color
-  };
+const Button = ({backgroundColor, borderRadius, clickData, color, onClick, text, type, width}) => (
```

Comment:
Props number has increased, it's visually better to write them on multiple lines instead of just one